contracts/Core/IBlockManager.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.5.10;
+pragma solidity 0.6.11;
 pragma experimental ABIEncoderV2;
```

Comment:
I think this line isn't required anymore in solidity 0.6. please check docs and remove if it's okay to.

Reply:
I have to add this line. Gives me the following error:

TypeError: This type is only supported in ABIEncoderV2. Use "pragma experimental ABIEncoderV2;" to enable the feature.
    function getBlock(uint256 epoch) external view returns(Structs.Block memory _block);

Reply:
https://solidity.readthedocs.io/en/v0.6.12/layout-of-source-files.html?highlight=abi%20encoder%20v2#abiencoderv2
Seems it's not experimental but you still have to add this line.